python/ray/tests/test_cli_logger.py

```diff
@@ -0,0 +1,17 @@
+from ray.autoscaler._private import cli_logger
```

Comment:
you'll need to add this to BUILD, but we can do that in a separate PR

Reply:
Added here, because of Lint failure :'(